安装make的时候报错

我make编译这一步，报错如下
go build -mod=vendor -ldflags "-X \"main.BuildVersion=bddc200\" -X \"main.BuildDate=2019-11-28 19:21:59\"" -o ./bin/kingshard ./cmd/kingshard
flag provided but not defined: -mod
usage: build [-o output] [-i] [build flags] [packages]
Run 'go help build' for details.
make: *** [kingshard] Error 2

是缺少啥嘛，我的安装步骤是按照文档中的来的
	1. 安装Go语言环境（Go版本1.6以上），具体步骤请Google。
	2. git clone https://github.com/flike/kingshard.git $GOPATH/src/github.com/flike/kingshard
	3. cd src/github.com/flike/kingshard
	4. source ./dev.sh
	5. make
	6. 设置配置文件
	7. 运行kingshard。./bin/kingshard -config=etc/ks.yaml

--- a/doc/KingDoc/how_to_use_kingshard.md
+++ b/doc/KingDoc/how_to_use_kingshard.md
@@ -133,7 +133,7 @@ schema_list :
 
 ### (2). 安装和启动kingshard
 
-1. 安装Go语言环境（Go版本1.3以上），具体步骤请Google。
+1. 安装Go语言环境（请使用最新版），具体步骤请Google。
 2. git clone https://github.com/flike/kingshard.git src/github.com/flike/kingshard
 3. cd src/github.com/flike/kingshard
 4. source ./dev.sh

--- a/doc/KingDoc/kingshard_install_document.md
+++ b/doc/KingDoc/kingshard_install_document.md
@@ -1,6 +1,6 @@
 ## 安装kingshard
 ```
-	1. 安装Go语言环境（Go版本1.6以上），具体步骤请Google。
+	1. 安装Go语言环境（请使用最新版），具体步骤请Google。
 	2. git clone https://github.com/flike/kingshard.git $GOPATH/src/github.com/flike/kingshard
 	3. cd src/github.com/flike/kingshard
 	4. source ./dev.sh

